requirements-dev.txt

```diff
@@ -12,4 +12,5 @@ decorator
 wheel
 
 # install flasgger itself as editable
--e .
\ No newline at end of file
+-e .
+-e git://github.com/Julian/jsonschema.git#egg=jsonschema
```

Comment:
Why add both in requirements and requirements-dev

Reply:
Because its not required

Reply:
Because dev requirements are only for dev related packages, that is why there are 2 seperate files